### ISSUE TITLE
retrieving point cloud after SfM from `sfm_data.bin`

### ISSUE THREAD

**Issue by muye1202** [openMVG/openMVG#2341]
Hi!

After I obtain the `sfm_data.bin`, I converted it to .json file. I see in the openmvg data structure wiki that `X` in structure -> value represents the 3D point coordinate, this is the result of the scene reconstruction right?

thanks

**Comment by pmoulon** [comment]
Yes it is. You can see what SFM data contains [here](https://github.com/openMVG/openMVG/wiki/OpenMVG-data-structures)

**Comment by pmoulon** [comment]
You can also convert to ply file directly
